file_formats/ie_formats/cre_v1.htm

```diff
@@ -484,11 +488,130 @@
             <td>Tracking skill (0-100)</td>
           </tr>
           <tr>
+            <td colspan="3"><b>The following entry applies to BG1, BG2 and BGEE:</b></td>
+          </tr>
+          <tr>
             <td>0x0084</td>
             <td>32 (char array)</td>
             <td>Tracking target</td>
           </tr>
           <tr>
+            <td colspan="3"><b>The following entries apply only to PSTEE:</b></td>
+          </tr>
+          <tr>
+            <td>0x0084</td>
+            <td>4 (dword)</td>
+            <td>Current thief class experience</td>
+          </tr>
+          <tr>
+            <td>0x0088</td>
+            <td>4 (dword)</td>
+            <td>Current mage class experience</td>
+          </tr>
+          <tr>
+            <td>0x008c</td>
+            <td>1 (byte)</td>
+            <td>GOOD variable increment value</td>
+          </tr>
+          <tr>
+            <td>0x008d</td>
+            <td>1 (byte)</td>
+            <td>LAW variable increment value</td>
+          </tr>
+          <tr>
+            <td>0x008e</td>
+            <td>1 (byte)</td>
+            <td>LADY variable increment value</td>
+          </tr>
+          <tr>
+            <td>0x008f</td>
+            <td>1 (byte)</td>
+            <td>Faction (see <a href="../../files/ids/pst/faction.htm">FACTION.IDS</a>)</td>
+          </tr>
+          <tr>
+            <td>0x0090</td>
+            <td>1 (byte)</td>
+            <td>Team (see <a href="../../files/ids/pst/team.htm">TEAM.IDS</a>)</td>
+          </tr>
+          <tr>
+            <td>0x0091</td>
+            <td>1 (byte)</td>
+            <td>Species (see <a href="../../files/ids/pst/race.htm">RACE.IDS</a>)</td>
+          </tr>
+          <tr>
+            <td>0x0092</td>
+            <td>1 (byte)</td>
+            <td>Dialogue activation range</td>
+          </tr>
+          <tr>
+            <td>0x0093</td>
+            <td>1 (byte)</td>
+            <td>Selection circle size</td>
+          </tr>
+          <tr>
+            <td>0x0094</td>
+            <td>1 (byte)</td>
+            <td>Shield flags:
+              <ul>
+                <li>Bit 0: Normal shield</li>
+                <li>Bit 1: Black barbed shield</li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <td>0x0095</td>
+            <td>1 (byte)</td>
+            <td>Field of vision</td>

```

Comment:
what unit is this in?

Reply:
I'm not even sure it measures in units. The field contains values between 0 and 8, with 4 being most frequent, followed by 8, 3, 0 and 1. Values 5..7 are not used at all. It could very well be a bitfield of some sort. I'll tag it as "unknown" until we know more.